### PR TITLE
gtk+3: update to 3.24.24.

### PR DESCRIPTION
**Pull request by ifreund**

### CODE REVIEW

File: srcpkgs/gtk-layer-shell/template [hunk 1]
```diff
@@ -1,9 +1,10 @@
 # Template file for 'gtk-layer-shell'
 pkgname=gtk-layer-shell
 version=0.5.1
-revision=1
+revision=2
 build_style=meson
 build_helper="gir"
+configure_args="-Dtests=true"
```
Comment (ifreund) [inline srcpkgs/gtk-layer-shell/template:7]: cc @travankor

Comment (ahesford) [reply srcpkgs/gtk-layer-shell/template:7]:
```bash
if [ "$XBPS_CHECK_PKGS" ]; then
	configure_args+=" -Dtests=true"
fi
```
is probably a better bet, so we only enable tests when somebody is actually running them.

**Review by ahesford — CHANGES_REQUESTED** [review]

File: srcpkgs/gtk-layer-shell/template (hunk 1 above)
Comment (ifreund) [reply srcpkgs/gtk-layer-shell/template]: Good call, pushed your suggestion

**Comment by ericonr** [comment]
I've been running this for a while now, with no issues.

**Pull request merged**